src/main/kotlin/com/codecraft/ai/api/engine/KeyFusionAPI.kt

```diff
@@ -0,0 +1,68 @@
+package com.codecraft.ai.api.engine
+
+import com.codecraft.ai.api.models.Text2ImageRequest
+import com.codecraft.ai.api.models.Text2ImageResponse
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.plugins.*
+import io.ktor.client.plugins.contentnegotiation.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import io.ktor.serialization.kotlinx.json.*
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import kotlinx.serialization.json.Json
+import java.nio.charset.StandardCharsets
+import java.util.*
+
+suspend fun startText2ImageProcess(call: ApplicationCall) {
+    call.request.apply {
+        val promptFromUser = queryParameters[QueryParam.PROMPT.strRepresentation] ?: return Error.EMPTY_PROMPT.let {
```

Comment:
moer than 100 cahracters here

Reply:
I do not understand what you mean here

Reply:
it needs to be reformated. Kotlin recommends line length of 100 chars

Reply:
aaa ok now got it